### ISSUE TITLE
You are right!

### ISSUE THREAD

**Issue by Mut1nyJD** [lucidrains/transframer-pytorch#1]
The only semi new thing is the DCT but even that I think is not completely new either. I thought there was already a generative model that did that not sure if it was autoregressive though.

But I guess the rational is that DCT is better than any learned quantized fixed length encoder and it saves you training an encoder.

**Comment by lucidrains** [comment]
Indeed! Regardless I'll finish the project so I can figure out if DCT representation has some sort of Achilles' heel
